tile.yml

```diff
@@ -1,28 +1,18 @@
 ---
-name: wavefront-nozzle # By convention lowercase with dashes
+name: wavefront-nozzle-glp
 icon_file: resources/wavefront.png
-label: Wavefront by VMware Nozzle for PCF
+label: Wavefront by VMware Nozzle for PCF GO
 description: Wavefront proxy and firehose nozzle
-metadata_version: 1.8                 # Optional, defaults to 1.5
+metadata_version: 1.8
 
-# Global defaults (all optional)
-#
-org: wavefront-nozzle-org                       # Name of org to create for your apps
-space: wavefront-apps-space                     # Name of space to create for your apps
-# apply_open_security_group: true               # Apply open security group, default: false
+org: wf-org-go 
+space: wf-apps-go
 
-# Specify the packages to be included in your tile.
-# The format of the section to include depends on the type
-# of package you are describing. For fragments of each type
-# that you can copy & paste, see:
-#
-# https://github.com/cf-platform-eng/tile-generator/blob/master/README.md
-#
 packages:
-- name: wavefront-proxy
+- name: wavefront_proxy
   type: bosh-release
   label: Wavefront Proxy
-  path: resources/proxy-bosh-release.tgz
+  path: resources/wf-proxy-bosh-release.tgz
```

Comment:
should revert to previous. We need to start checking in the `dev_releases` build folder for the bosh release. That will allow to continue building new bosh releases without running into versioning issues.